blocks/carousel/carousel.css

```diff
@@ -0,0 +1,65 @@
+main .carousel-container {
```

Comment:
is `main` necessary in the selector ?

Reply:
I don't think so, as I said, I mostly put in some styling to make a point about how I was thinking it could work. Let me remove the main.